build_tools/bazel/workspace.bzl

```diff
@@ -8,6 +8,63 @@
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("@bazel_skylib//lib:paths.bzl", "paths")
 
+CUDA_TOOLKIT_ROOT_ENV_KEY = "IREE_CUDA_TOOLKIT_ROOT"
+
+# See base docker image: We place a stripped down CUDA directory tree
+# here if this env var is set on CI.
```

Comment:
Agreed on it being a followup. The issue is that the CMake CUDA finding automation does not recognize the stripped down toolkit (it keys off of certain files that only come with an official install, iirc). This can probably be simplified but was trying to not get too adventurous in one go. I updated the comment and added a TODO.